tools/otsdb-dump-restore/README.md

```diff
@@ -0,0 +1,146 @@
+                              dumped  [default: kilda.]
+  --remove-metadata
+  --help                      Show this message and exit.
+```
+
+### Restore data to OpenTSDB:
+Use `kilda-otsdb-restore` command to restore data previously dumped to an OpenTSDB.
+```
+Usage: kilda-otsdb-restore [OPTIONS] OPENTSDB_ENDPOINT
+
+  This tool restore the data to an OpenTSDB
+
+  OPENTSDB_ENDPOINT openTSDB endpoint
+
+  Example: kilda-otsdb-restore http://example.com:4242
+
+Options:
+  --dump-dir DIRECTORY          Location where dump files are stored
+  --request-size-limit INTEGER  Limit for "put" request payload size (bytes)
+                                [default: 4096]
+  --help                        Show this message and exit.
+```
+### Date and time formats
+```
+%Y-%m-%d, %Y-%m-%dT%H:%M:%S, %Y-%m-%d %H:%M:%S
+```
+- "2023-03-22"
+
+- "2023-03-22T23:59:59"
+
+- "2023-03-22 23:59:59"
+
+### Example of use:
+Scenario:
+* One OpenTSDB service that has stored data.
+* One empty VictoriaMetrics service with OpenTSDB capabilites.
+* We want to migrate the data from the OpenTSDB to the VictoriaMetric service.
+
+This could be a posible workflow:
+1. Create volume to store data
+```bash
+docker volume create opentsdb-data
+```
+2. Next, we need to dump the data from OpenTSDB. The following command gets the data from a specified time until now and save it to disk:
+```bash
+docker run --rm -v opentsdb-data:/tmp kilda-otsdb-dump-restore kilda-otsdb-dump http://opentsdb:4242 "2023-03-08"
+```
+3. After dumping the data, we can restore it using the following command:
+```bash
+docker run --rm -v opentsdb-data:/tmp kilda-otsdb-dump-restore kilda-otsdb-restore http://victoria:4242
+```
+4. Finally, we can remove the volume using the following command:
+```bash
+docker volume remove opentsdb-data
+```
+
+Another approach would be to use a loop that iterates over a time range, such as days. This method could be beneficial when migrating a large amount of data.
+1. Create volume to store data
+```bash
+docker volume create opentsdb-data
+```
+2. Next, we need to dump the data from OpenTSDB. Use the following command to get the data from a specified time until the --time-stop time and save it to disk:
+```bash
+docker run --rm -v opentsdb-data:/tmp kilda-otsdb-dump-restore kilda-otsdb-dump --time-stop "2023-03-22T11:00:00" http://opentsdb:4242 "2023-03-22T00:00:00"
+```
+3. After dumping the data, we can restore it using the following command:
+```bash
+docker run --rm -v opentsdb-data:/tmp kilda-otsdb-dump-restore kilda-otsdb-restore http://victoria:4242
+```
+4. Finally, we can remove the volume using the following command:
+```bash
+docker volume remove opentsdb-data
+```
+
+### How to use the `otsdb-to-vm.sh` script
+The `otsdb-to-vm.sh` script is a bash wrapper around the `kilda-otsdb-dump-restore` tool. It is used to dump data from an OpenTSDB and restore it to a VictoriaMetrics service.
```

Comment:
```suggestion
The `otsdb-to-vm.sh` script is a bash wrapper around the `kilda-otsdb-dump-restore` tool. It is used for dumping data from an OpenTSDB and restore it to a VictoriaMetrics service.
```
`is used to` is a special collocation that means "get familiar with`, without to be it means "did something in the past, but not now"